Tutorials/Sprite-Related-Tutorials/Sprite Layering Tutorial/01-02-sprites-layering.mdx

```diff
@@ -47,56 +69,118 @@ With our code at this stage, we basically just have a single-layered `Sprite`. U
 cases, but let's say we want to have user input, or specific functionality of the game "show" layers
 of a singular `Sprite`? Well, using our already-defined bitmaps, it's relatively simple.
 
+
+<Tabs>
+  <TabItem label="C#">
 ```csharp
 SplashKit.SpriteAddLayer(bodySprite, hatBit, "hat");                        // layer 1
 SplashKit.SpriteAddLayer(bodySprite, starBit, "star");                      // layer 2
 ```
+  </TabItem>
+  <TabItem label="C++">
+```c++
+SplashKit.SpriteAddLayer(bodySprite, hatBit, "hat");                        // layer 1
+SplashKit.SpriteAddLayer(bodySprite, starBit, "star");                      // layer 2
```

Comment:
C# code here, you'll likely want 
```
sprite_add_layer(bodySprite, hatBit, "hat");
sprite_add_layer(bodySprite, starBit, "star");
```
See: (https://splashkit.io/api/sprites/#sprite-add-layer)